web/satellite/src/views/Billing.vue

```diff
@@ -112,7 +112,6 @@
                                 <v-divider class="my-4" />
 
                                 <v-btn
-                                    v-if="couponCodeBillingUIEnabled"
```

Comment:
IIRC the idea of this config was so users can not add coupons via the UI. Has that changed?

Reply:
oops, i was testing something and forgot this, thanks Wilfred!